From: Julian Ospald <hasufell@gentoo.org>
Date: Tue Jun  5 23:20:10 UTC 2012
Subject: build system

fix linking for linkers that don't allow underlinking
applied on top of cubetrains-${PV}-build.patch

--- Makefile
+++ Makefile
@@ -14,14 +14,16 @@
 	    -I/usr/include/boost \
 	    -I/sw/include/SDL \
 	    -Isrc/
-LIBS += -lSDLmain \
+LIBS += $(shell pkg-config --libs x11) \
+	-lSDLmain \
 	$(shell pkg-config --libs sdl) \
 	$(shell pkg-config --libs glu) \
 	$(shell pkg-config --libs glew) \
 	$(shell pkg-config --libs SDL_image) \
 	$(shell pkg-config --libs SDL_ttf) \
 	$(shell pkg-config --libs SDL_mixer) \
-	$(shell pkg-config --libs libpng)
+	$(shell pkg-config --libs libpng) \
+	$(shell pkg-config --libs zlib)
 
 LDFLAGS += -L/sw/lib \
 	   -L/usr/lib \
